salt/_states/metalk8s.py

```diff
@@ -0,0 +1,32 @@
+"""Custom states for MetalK8s."""
+
+__virtualname__ = "metalk8s"
+
+
+def __virtual__():
+    return __virtualname__
+
+
+def static_pod_managed(name, source, config_files, context):
+    """Simple helper to edit a static Pod manifest if configuration changes.
+
+    Expects the template to use the `config_digests` variable and store it in
+    the `metadata.annotations` section, with the key
+    `metalk8s.scality.com/config-digests`.
+    """
+    config_digests = [
+        __salt__["hashutil.digest_file"](config_file, checksum="sha256")
+        for config_file in config_files
+    ]
+
+    return __states__["file.managed"](
+        name,
+        source,
+        template="jinja",
+        user="root",
+        group="root",
+        mode="0644",
```

Comment:
Maybe this should be `0600`? No need to be world-readable.

Reply:
Ok will do